idunn/api/instant_answer.py

```diff
@@ -323,9 +327,25 @@ async def fetch_pj_response():
             )
             return build_response(result, query=q, lang=lang)
 
-    # Call tripadvisor datasource instant answer outside France or
+    # Get osm if result with wikidataID or tripadvisor datasource instant answer outside France or
     # without intention location detection
     else:
+        bragi_osm_response = await fetch_bragi_osm
+        if bragi_osm_response["features"]:
+            place_id = bragi_osm_response["features"][0]["properties"]["geocoding"]["id"]
+            try:
+                place = place_from_id(place_id, lang, follow_redirect=True)
+            except Exception:
+                logger.warning(
+                    "get_instant_answer: Failed to get place with id '%s'", place_id, exc_info=True
+                )
+                return no_instant_answer()
+            if place.wikidata_id and place.properties.get("poi_subclass", True) not in [
```

Comment:
I'm not sure why `True` is the default value in `place.properties.get("poi_subclass", True)`? Leaving the default `None` would be sufficient?

Reply:
The default value is used to validate the condition for `admin` that don't have `poi_sublass` key in their dict. I am adding a comment because it is totally implicit.